services/auth/src/monitor.js

```diff
@@ -0,0 +1,17 @@
+import { MonitorManager } from 'taskcluster-lib-monitor';
```

Comment:
This is probably fine to keep a copy in every service, but we can also define it once in a single `builtins` place - https://github.com/taskcluster/taskcluster/blob/28c7af13a54a4309bab0f8b92a81a9d4d9c9b97d/libraries/monitor/src/builtins.js